<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TelaCadastroGerente</class>
 <widget class="QFrame" name="TelaCadastroGerente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>374</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>380</width>
    <height>374</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>380</width>
    <height>374</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Gerente</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>177</x>
     <y>330</y>
     <width>191</width>
     <height>36</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Cancelar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Confirmar</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Unidade</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_8"/>
  </widget>
  <widget class="QSplitter" name="splitter_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_7">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Senha</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_7"/>
  </widget>
  <widget class="QSplitter" name="splitter_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_6"/>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Sal√°rio</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_5"/>
  </widget>
  <widget class="QSplitter" name="splitter_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Endereco</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_4"/>
  </widget>
  <widget class="QSplitter" name="splitter_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Telefone</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_3"/>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>CPF</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2"/>
  </widget>
  <widget class="QSplitter" name="splitter_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>13</y>
     <width>361</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="minimumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>70</width>
      <height>30</height>
     </size>
    </property>
    <property name="text">
     <string>Nome</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit"/>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
